constructor_io/modules/catalog.py

```diff
@@ -413,25 +416,21 @@ def retrieve_variations(self, parameters=None):
         '''
         Retrieves multiple variations from the index (limit of 1,000)
 
-        :param list parameters.ids: A list of item IDs to retrieve
-        :param str parameters.section: The section to update
-        :param int parameters.num_results_per_page: The number of variations to return. Defaults to 100. Maximum value 100
+        :param list parameters.ids: A list of variation IDs to retrieve
+        :param str parameters.item_id: Item ID of variations to retrieve
+        :param str parameters.section: The section to retrieve from
+        :param int parameters.num_results_per_page: The number of variations to return. Defaults to 100. Maximum value 1000
```

Comment:
Same as above, it says 100 in the [docs](https://docs.constructor.io/rest_api/items/variations/#url-parameters-2) 🤔 Are the docs incorrect?

Reply:
Nope I just read the wrong line 😂